### PR TITLE
chore: improve happo screenshot names for cypress

### PR DESCRIPTION
**Pull request by ozgurkececioglu**
[FX-2850]

### Description

PR is about giving better names for happo screenshots. It is important to group the components on Picasso/Cypress happo reports

### How to test

- checking variant names' corrections would be enough

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| Screen Shot 2022-06-24 at 10 31 14 | waiting for happo report |

### Development checks

- Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- Annotate all `props` in component with documentation
- Create `examples` for component
- Ensure that deployed demo has expected results and good examples
- Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [ ] Self reviewed
- Covered with tests

**Breaking change**

- codemod is created and showcased in the changeset
- test alpha package of Picasso in StaffPortal

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-2850]: https://toptal-core.atlassian.net/browse/FX-2850?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: cypress/component/BarChart.spec.tsx [hunk 1]
```diff
@@ -52,11 +52,16 @@ const assertCustomTooltipContent = (text: string) => {
...
 describe('BarChart', () => {
   it('renders default chart with default tooltip on hover', () => {
     cy.mount(<TestBarChart />)
 
-    hoverOverBar('Apple').get('body').happoScreenshot()
+    hoverOverBar('Apple').get('body').happoScreenshot({
+      component,
+      variant: 'default-chart/default-tooltip',
+    })
     assertTooltipContent('Appleengineers hired : 500')
```
Comment (separatio) [inline cypress/component/BarChart.spec.tsx:65]: `Appleengineers` vs `Apple engineers` :smile: Just a nit. I find it both annoying and funny.

File: cypress/component/Badge.spec.tsx [hunk 2]
```diff
@@ -32,7 +37,10 @@ describe('Badge', () => {
           </Container>
         </Container>
       )
-      cy.get('body').happoScreenshot()
+      cy.get('body').happoScreenshot({
+        component,
+        variant: 'white/small-size',
```
[the remark this answers is not in the document]
Comment (separatio) [reply cypress/component/Badge.spec.tsx:42]: Alternatively, we can do an `all sizes` screenshot, like for ButtonAction. However, I feel that this should have been covered in Storybook. :smiling_face_with_tear:

File: cypress/component/Table.spec.tsx [hunk 3]
```diff
@@ -281,71 +281,106 @@ type Data = {
   defaultExpanded: boolean
 }
 
+const component = 'Table'
```
Comment (separatio) [inline cypress/component/Table.spec.tsx:284]: I think most examples for this component are already in Storybook, visually tested.

**Review by separatio — APPROVED** [review]
The renames are good generally. Only had a few nits basically. It might be great to remove tests duplicated by Storybook first, though. I feel like we're duplicating work and review time from our team.

File: cypress/component/Table.spec.tsx (hunk 3 above)
Comment (ozgurkececioglu) [reply cypress/component/Table.spec.tsx:284]: yup, felt the same.

**Review by denieler — APPROVED** [review]
Oh, wow, that's huge 💪 but amazing work 👍

**Comment by ozgurkececioglu** [comment]
I'm closing this PR because we already decided to separate the work into 6 different PRs.

**Pull request closed without merging**